### DISCUSSION TITLE
Autofill lowercase or snake case Path as per DOCS?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#10315] [Editor]
DOCS [https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html](https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html) :

"Windows and recent macOS versions use case-insensitive filesystems by default, whereas Linux distributions use a case-sensitive filesystem by default. This can cause issues after exporting a project, since Godot's PCK virtual filesystem is case-sensitive. To avoid this, **it's recommended to stick to snake_case naming for all files in the project (and lowercase characters in general).**"

When adding a script to a node the popup autofills Path with the node name, which usually contains uppercase letters, which goes against the docs.

To be consistent either Path field which generates the file name should be snake cased automagically or all lowercased, right?

Thoughts?

**Comment by AThousandShips** [comment]
This is already the default, new scripts default to the file casing setting, which defaults to `snake_case`, this is a duplicate of:
* https://github.com/godotengine/godot-proposals/issues/1211

Which was implemented in:
* https://github.com/godotengine/godot/pull/78119

**Reply by shak2** [reply]
Reading that page, does it mean it is now implemented starting at 4.3? (I am still 4.2)

**Reply by AThousandShips** [reply]
Yes, it's a new feature in 4.3, and has been implemented, as it is a new feature it won't make it to older versions
